Frontend/ui-driver/src/Flow.purs

```diff
@@ -673,7 +673,7 @@ onBoardingFlow = do
   void $ pure $ hideKeyboardOnNavigation true
   config <- getAppConfigFlowBT Constants.appConfig
   GlobalState allState <- getState
-  DriverRegistrationStatusResp driverRegistrationResp <- driverRegistrationStatusBT $ DriverRegistrationStatusReq { }
+  DriverRegistrationStatusResp driverRegistrationResp <- driverRegistrationStatusBT $ DriverRegistrationStatusReq "?makeSelfieAadhaarPanMandatory=true"
```

Comment:
handle query-param inside endpoint and pass it as argument to `driverRegistrationStatusBT`